ssv/proposer.go

```diff
@@ -314,7 +327,11 @@ func (r *ProposerRunner) GetBeaconNode() BeaconNode {
 }
 
 func (r *ProposerRunner) GetShare() *types.Share {
-	return r.BaseRunner.Share
+	// there is only one share
+	for _, share := range r.BaseRunner.Share {
+		return share
+	}
```

Comment:
honestly its kinda hacky that now all runners have share map just to support committee runnner

Reply:
@GalRogozinski 
When we consolidate all messages to use CommitteeID instead of validator pubkey, this hack can be removed

The other solution of leaving validator pubkey would require heavy risky refactoring to spec